src/Tokenizer/Tokens.php

```diff
@@ -358,7 +363,7 @@ public function clearChanged(): void
      */
     public function clearEmptyTokens(): void
     {
-        $limit = $this->count();
+        $limit = \count($this);
```

Comment:
array-like calls are faster on SplFixedArray

- https://3v4l.org/O2nWA/rfc#vgit.master
- https://3v4l.org/BrkZm/rfc#vgit.master_jit

Reply:
I wonder if we can somehow enforce it on CI level, so we don't have another PR that would reintroduce ->count()

Reply:
Custom PHPStan rule.